.github/workflows/formatting.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup toolchain
-        uses: dtolnay/rust-toolchain@1.66.0
+        uses: dtolnay/rust-toolchain@1.67.0
```

Comment:
Please say more why version bump?

Reply:
1.66.1 doesn't build WASI correctly. It includes a `__errno_location` import that doesn't exist in the WASI spec. 1.67 fixes that.